### ISSUE TITLE
Invalid parent selector :not(.disabled) in sass file

### ISSUE THREAD

**Issue by Eldar-X** [quasarframework/quasar#6194]
**Describe the bug**

```
Module build failed (from /Users/user/project/node_modules/sass-loader/dist/cjs.js):
SassError: Invalid parent selector ":not(.disabled)"
     ╷
4347 │ ┌     &--dense:focus,
4348 │ └     &--dense:hover
```

Looks like because parent selector have additional selector ":not" &--dense can't be applied.

**Platform (please complete the following information):**
OS: Macos
Node: 13.5
NPM: 
Yarn: 1.21.1
Browsers:
iOS:
Android:
Electron:

**Additional context**

**Comment by rstoenescu** [comment]
Hi,

Thanks for reporting!
Fix will be available in "quasar" v1.8.1 in short time.

**Comment by rstoenescu** [comment]
v1.8.1 is now available and contains the fix.
